udata_recommendations/tasks.py

```diff
@@ -0,0 +1,16 @@
+from flask import current_app
+
+from udata.tasks import job
+
+from .commands import process_sources, log as commands_log
+
+
+log = commands_log
```

Comment:
Is it fine to relay logs this way?

Reply:
Not sure this is a good idea. I'd instantiate a new one. But you're not using it anyway?